### PR TITLE
Improved Footer Design

### PR DESCRIPTION
**Pull request by Priyanshu-official**
Improved Footer Design #470 
Used normal CSS and Bootstrap

Design in black mode

footer

### CODE REVIEW

**Comment by Priyanshu-official** [comment]
@smaranjitghose Pls Review PR and let me Know any changes needed

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5502
```
Comment (smaranjitghose) [inline .vscode/settings.json:2]: Remove this and add it to .gitignore please

**Comment by smaranjitghose** [comment]
responsiveee
There is an issue with responsiveness. Could you please fix this^^

File: assets/css/style.css [hunk 2]
```diff
@@ -289,344 +289,199 @@ html.transition *:after {
 
 /*Footer*/
```
Comment (smaranjitghose) [inline assets/css/style.css:290]: Make District Comments like

/*-----------------
FOOTER SECTION
-------------------*/

File: assets/css/style.css [hunk 3]
```diff
@@ -289,344 +289,199 @@ html.transition *:after {
...
+.footer{
+  background-color: black;
 }
 
-.row {
-  clear: both;
-  margin: 0 auto;
-  padding: 0 auto;
-  bottom: 0px;
-  clear: both !important;
```
Comment (smaranjitghose) [inline assets/css/style.css:303]: Add  relevant comments

File: assets/css/style.css [hunk 4]
```diff
@@ -289,344 +289,199 @@ html.transition *:after {
...
-  flex: 1;
-  justify-content: space-between;
-}
-.footer-left {
-  display: flex;
-  flex-direction: column;
-  width: 59%;
-  margin: auto;
-  margin-left: -4%;
+.rounded-social-buttons .social-button.twitter:hover, .rounded-social-buttons .social-button.twitter:focus {
```
Comment (smaranjitghose) [inline assets/css/style.css:347]: Add Relevant comments

File: assets/css/style.css [hunk 5]
```diff
@@ -289,344 +289,199 @@ html.transition *:after {
...
-  margin-top: 0px;
-  margin-bottom: 0px;
-}
-.webintern_footer_title small {
-  font-family: "Work Sans", sans-serif;
-  color: #fff;
-  font-weight: 500;
-  font-size: 10px;
-  text-transform: uppercase;
+/* FORM CSS*/
```
Comment (smaranjitghose) [inline assets/css/style.css:402]: Add meaning full comments like "Contact Form in Footer"

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

**Comment by Priyanshu-official** [comment]
Ok💯

**Comment by Priyanshu-official** [comment]
@smaranjitghose Sir Can pls Review this I check Responsive is Working fine for me

MG1

IP1

**Comment by Priyanshu-official** [comment]
@smaranjitghose If everything is fine then Pls added both Label DWOC and SWOC if possible

**Review by anushbhatia — CHANGES_REQUESTED** [review]
Transitions in GitHub is really weird and it moves to the right of it. Add a minimal hover effect on it

**Comment by smaranjitghose** [comment]
@Priyanshu-official  Neither the social media icons nor the quote in the footer is centre aligned in the mobile view

**Comment by smaranjitghose** [comment]
Also, can you pull the current changes from master and incorporate them

**Comment by Priyanshu-official** [comment]
@smaranjitghose sorry I didn't notice they are not center I will do it

**Comment by Priyanshu-official** [comment]
@smaranjitghose Pls Review PR I have fixed all Responsive issue

**Review by anushbhatia — CHANGES_REQUESTED** [review]
Create a new PR because files have been modified
Add footer in app.js

**Comment by anushbhatia** [comment]
color palllete

**Comment by Priyanshu-official** [comment]
@anushbhatia Ok Is Everything is fine in the footer Just I have to move the code to the app.js File Right??

**Pull request closed without merging**